examples/e18_webhook/src/main.rs

```diff
@@ -0,0 +1,13 @@
+use serenity::http::HttpBuilder;
+
+#[tokio::main]
+async fn main() {
+    // You don't need a token when you are only dealing with webhooks
+    let http = HttpBuilder::new("").build();
+    let webhook = http.get_webhook_from_url("https://discord.com/api/webhooks/133742013374206969/hello-there-oPNtRN5UY5DVmBe7m1N0HE-replace-me-Dw9LRkgq3zI7LoW3Rb-k-q").await.expect("Replace the webhook with your own");
```

Comment:
```suggestion
    let webhook = Webhook::from_url(&http, "https://discord.com/api/webhooks/133742013374206969/hello-there-oPNtRN5UY5DVmBe7m1N0HE-replace-me-Dw9LRkgq3zI7LoW3Rb-k-q").await.expect("Replace the webhook with your own");
```

More ergonomic to construct the Webhook directly. Will need to import it as well.